routers/private/hook_post_receive.go

```diff
@@ -105,6 +106,8 @@ func HookPostReceive(ctx *gitea_context.PrivateContext) {
 				}
 			} else {
 				branchesToSync = append(branchesToSync, update)
+
+				pull_service.UpdatePullsRefs(ctx, repo, update)
```

Comment:
The errors are just logged but do not block further processing. Is this intended?

Reply:
Yes. Keeping this PR has the same logic as before. Maybe we can have further discussion and change it in future.

Reply:
Add a comment and TODO ?